lms/product/__init__.py

```diff
@@ -5,6 +5,14 @@
 
 
 def includeme(config):
+    # Register the default plugins
+    config.include("lms.product.plugin")
+
+    # Give everyone a chance to register their plugins etc.
+    config.include("lms.product.canvas")
+    config.include("lms.product.blackboard")
```

Comment:
Little bit of make work here to register each new product, but literally a one liner